notebook/converted/batch_effect_correction.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Evaluation of Batch Effect Correction
+# This notebook asks whether batch effect correction works for correcting inter-study differences and/or interplatform differences.
+
+# In[1]:
+
+
+import collections
+import json
+import os
+import pickle
+from typing import Text, Dict, Union, List
+
+import pandas as pd
+import sklearn
+from plotnine import ggplot, geom_point, aes
+from tqdm.notebook import tqdm
+
+from saged import utils
+
+
+# In[2]:
+
+
+def return_unlabeled():
```

Comment:
`defaultdict`s take in a callable that provides a default value instead of the value itself. I could use `sample_to_label = collections.defaultdict(lambda: "unlabeled", sample_to_label)` to do the same thing, but I felt like creating a small function was clearer